iocore/net/SSLUtils.cc

```diff
@@ -544,7 +547,64 @@ ssl_context_enable_ecdh(SSL_CTX *ctx)
 
   return ctx;
 }
+static ssl_ticket_key_block *
+ssl_create_ticket_keyblock(const char *ticket_key_path)
```

Comment:
Hmmm, maybe I'm missing something, but it seems most of this code is duplicated from ssl_context_enable_tickets() ? Can we not refactor this such that we don't get such massive code duplication?